test/oranif_eunit.erl

```diff
@@ -696,6 +696,67 @@ stmtGetQueryInfoFail(#{session := Conn} = TestCtx) ->
     ),
     dpiCall(TestCtx, stmt_close, [Stmt, <<>>]).
 
+stmtGetInfo(#{session := Conn} = TestCtx) ->
```

Comment:
See if this test can be removed now!
Since `stmtGetInfoStmtTypes` is already covering the same